server/src/main/java/com/linecorp/centraldogma/server/metadata/MetadataService.java

```diff
@@ -868,31 +868,64 @@ public CompletableFuture<Revision> destroyToken(Author author, String appId) {
         requireNonNull(author, "author");
         requireNonNull(appId, "appId");
 
-        // Remove the token from every project.
+        return tokenRepo.push(INTERNAL_PROJECT_DOGMA, Project.REPO_DOGMA, author,
+                              "Delete the token: " + appId,
+                              () -> tokenRepo
+                                      .fetch(INTERNAL_PROJECT_DOGMA, Project.REPO_DOGMA, TOKEN_JSON)
+                                      .thenApply(tokens -> {
+                                          final JsonPointer deletionPath =
+                                                  JsonPointer.compile("/appIds" + encodeSegment(appId) +
+                                                                      "/deletion");
```

Comment:
If a token is destroyed, do we prevent users from activating the token by 
https://github.com/line/centraldogma/blob/21eb395ce45f4e90a88a6a4cd38d411b574d5da2/server/src/main/java/com/linecorp/centraldogma/server/internal/api/TokenService.java#L174-L177